language-support/qute/qute.tmLanguage.json

```diff
@@ -400,15 +400,23 @@
           "name": "keyword.operator.arithmetic.java"
         },
         {
-          "match": "(!|&&|\\|\\|)",
+          "match": "(!)",
           "name": "keyword.operator.logical.java"
         },
         {
           "match": "(\\||&)",
           "name": "keyword.operator.bitwise.java"
         },
         {
-          "match": "\\b(is|as|eq|ne|gt|ge|lt|le)\\b",
+          "match": "(?<=({#if\\s(\\w)*\\s))(gt|>|ge|>=|lt|<|le|<=)",
```

Comment:
If you want to split match per category (logical, etc), you should change the name, otherwise use the same patch.

But I think  we should have `eq, is, ne, and, or` in `keyword.control.qute` and the other(`!= , ||, etc)`  should be `keyword.operator.qute` to have different color between them.

Reply:
As i zxplained we should not have the same color for or and ||. It means that or must be defined a given pattern and || in an another pattern